.github/workflows/create-signing-events.yml

```diff
@@ -0,0 +1,18 @@
+name: TUF-on-CI create Signing events
+
+on:
+  schedule:
+    - cron:  '17 1,7,13,19 * * *'
```

Comment:
Just wondering, what's the reason for this cadence?

Reply:
no specific reason: just something that runs multiple times a day (I went with 4) to be more often than the granularity of the signing period length (which is counted in days)